server/src/logic.js

```diff
@@ -80,11 +87,18 @@ export const getHandlers = ({ models, creators: Creators, push, pubsub }) => {
       updateUserProfile(_, args, ctx) {
         // TODO support more basic fields
         const { displayName } = args.user;
-        return getAuthenticatedUser(ctx).then(user =>
+        return getAuthenticatedUser(ctx).then((user) => {
           user.update({
```

Comment:
make sure you return this since you changed it to a block from a lambda

Reply:
thanks. nice catch